src/DynamoCore/Configuration/StyleItem.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.Configuration
```

Comment:
If the `HexColorString` would apply to all styles items but `GroupName` would only apply to group style items, should we make a base class first and then have a `GroupStyleItem` inheriting that? Just one approach to consider here, what do you think?

Reply:
I agree with that. If it's likely that we are going to serialize other styles in the future, we should specify these fields better.

Reply:
What do you think about renaming the class to `GroupStyleItem`?

Reply:
I am considering creating a class called `StyleItem` and have `GroupStyleItem` inheriting from that. Then we should be able to keep the list `StyleItemsList<StyleItem>`, hopefully it will work